generic/tests/cfg/ethtool.cfg

```diff
@@ -1,15 +1,20 @@
 - ethtool:
     virt_test_type = qemu libvirt
     only Linux
-    only e1000, virtio_net
+    only e1000, e1000e, virtio_net
```

Comment:
`only e1000 e1000e virtio_net`

Reply:
Sorry, my fault, I see `only one,three`

Reply:
`only e1000,e1000e,virtio_net`

Reply:
@sitoliu It's okay to have a space after `,`